scripts/release-asm/common.sh

```diff
@@ -236,3 +243,112 @@ setup() {
     touch asmcli
   fi
 }
+
+prompt_for_confirmation() {
+  local CONFIRMATION
+  read -r -p "${1} [y/N] " CONFIRMATION
+  case "${CONFIRMATION}" in
+      [yY][eE][sS]|[yY])
+          return
+          ;;
+  esac
+  false
+}
+
+prompt_for_message() {
+  local MSG
+  read -r -p "Please provide a release message for ${1}:" MSG
+  if [[ -n "${MSG}" ]]; then
+    echo "${MSG}"
+  fi
+}
+
+update_config_number() {
+  local OLD_NUMBER; OLD_NUMBER="${1}"
+  local NEW_NUMBER; NEW_NUMBER="${2}"
+
+  sed -i.bak "s/CONFIG_VER:=\"${OLD_NUMBER}\"/CONFIG_VER:=\"${NEW_NUMBER}\"/g" "${SCRIPT_PATH}.sh"
+
+  bazel build //:merge && cp ../bazel-bin/asmcli .
```

Comment:
I think maybe you meant to use the pre-release here?

Reply:
the `precommit` script assumes it's at the package root so I have to re-write the steps here.